utils/misc.mli

```diff
@@ -118,8 +139,9 @@ val replace_substring: before:string -> after:string -> string -> string
            occurences of [before] with [after] in [str] and returns
            the resulting string. *)
 
-val rev_split_words: string -> string list
-        (* [rev_split_words s] splits [s] in blank-separated words, and return
+val rev_split_words: ?separator:char -> string -> string list
```

Comment:
I think it's fine with a single function.  It's standard (e.g. in bash) for the notion of "words" be attached to that of a separator character, which has a default.